typings/index.test-d.ts

```diff
@@ -103,15 +102,6 @@ const client: Client = new Client({
     MessageManager: 200,
     // @ts-expect-error
     Message: 100,
-    ThreadManager: {
-      maxSize: 1000,
-      keepOverLimit: (x: ThreadChannel) => x.id === '123',
-      sweepInterval: 5000,
-      sweepFilter: LimitedCollection.filterByLifetime({
-        getComparisonTimestamp: (x: ThreadChannel) => x.archiveTimestamp ?? 0,
-        excludeFromSweep: (x: ThreadChannel) => !x.archived,
-      }),
-    },
```

Comment:
Shouldn't we keep this to validate the object overload without `sweep*` options?

Reply:
whoops, yeah, meant to add the new example here